tokio/src/sync/notify.rs

```diff
@@ -22,7 +22,7 @@ use std::task::{Context, Poll, Waker};
 ///
 /// The synchronization details of `Notify` are similar to
 /// [`thread::park`][park] and [`Thread::unpark`][unpark] from std. A [`Notify`]
-/// value contains a single permit. [`notfied().await`] waits for the permit to
+/// value contains a single permit. [`notified().await`] waits for the permit to
```

Comment:
See also  #2351.

Reply:
Still don't know how this was added. Anyway it's removed.
Thanks!